### ISSUE TITLE
Cell order changed when reload data

### ISSUE THREAD

**Issue by CocaCola183** [chiahsien/CHTCollectionViewWaterfallLayout#120]
When I reload, I found the cell's order changed, how can I keep the original order of cell?

**Comment by chiahsien** [comment]
Are cells' heights changed?

**Comment by emjoseph** [comment]
Yeah I'm getting the same behavior. Cell heights have not changed. Works correctly the first time reloadData is called, but heights reversed on subsequent calls

**Comment by emjoseph** [comment]
Also I'm running the Swift 3 version

**Comment by emjoseph** [comment]
@CocaCola183 were you able to resolve this issue on your own?

**Comment by emjoseph** [comment]
Was able to resolve the issue. Not a problem with the library but the way I was handling the reuse of my cells. The collectionViewCell's contentView frame was not updating on reuse hence the perceived reverse order

**Comment by hai2014bk** [comment]
@emjoseph how did you solve it ?

**Comment by mikengyn** [comment]
anyone solve this? I have the same issue
